herbert_core/utilities/misc/hlp_serialize.m

```diff
@@ -11,17 +11,17 @@
 %   Bytes : a representation of the original data as a byte stream
 %
 % Notes:
-%   The code is a rewrite of Tim Hutt's serialization code. Support has been added for correct 
-%   recovery of sparse, complex, single, (u)intX, function handles, anonymous functions, objects, 
+%   The code is a rewrite of Tim Hutt's serialization code. Support has been added for correct
```

Comment:
I know this is previous code, but it may be worth taking the opportunity to explain its origin - who is Tim Hutt, where did this code come from, who did the rewrite

Reply:
The answer to all of those questions is no idea. Best bet is to ask @abuts .